src/Features/Core/Portable/SolutionEvents/ISolutionEventsService.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.SolutionEvents
+{
+    internal interface ISolutionEventsService
```

Comment:
individual service that will hear about tehse events.  The UnitTestingSolutionCrawler will be one of these listeners.

Reply:
I'd call this `ILegacySolutionEventsListener` -- "legacy" as we do not want newly designed services to use this, and "listener" is more specific than "service".

Reply:
done.